0200-data-collection-field-navigation.Rmd

```diff
@@ -11,8 +11,20 @@ When in the field, we use Mergin Maps on our phones to visual the area, but also
 
 1. [Timestamp Camera](http://www.timestampcamera.com/) is used to take photos (landscape only) in the field. Ensure the current date and time and UTM coordinates are in top right corner using the settings below. 
 
+```{r timestamp, fig.cap = "Navigate to the Timestamp settings by pressing the "clock" icon in the bottom right corner",eval=T}
+
+# fpr::fpr_photo_resize_convert(photo = "/Users/lucyschick/Projects/repo/fish_passage_guidebook/fig/fieldwork/timestamp_settings_access.jpg", path = "/Users/lucyschick/Projects/repo/fish_passage_guidebook/fig/fieldwork/timestamp_settings_access.jpg")
+
+knitr::include_graphics("fig/fieldwork/timestamp_settings_access.jpg")
+
+```
+
 ```{r timestamp, fig.cap = "Timestamp settings",eval=T}
-knitr::include_graphics("fig/fieldwork/timestamp.png")
+
+# fpr::fpr_photo_resize_convert(photo = "/Users/lucyschick/Projects/repo/fish_passage_guidebook/fig/fieldwork/timestamp_settings.jpg", path = "/Users/lucyschick/Projects/repo/fish_passage_guidebook/fig/fieldwork/timestamp_settings.jpg")
```

Comment:
path is to the directory vs the photo. it uses same name

```
fpr::fpr_photo_resize_convert(photo = "/Users/airvine/Projects/repo/fish_passage_guidebook/fig/mapping/themes.png",
                             path = "fig/mapping")
```

![image](https://github.com/user-attachments/assets/e743b608-db51-4e9b-bd1b-5deb0732bbf0)


maybe you are already doing it but good way to view the phone is through the mac then take a screenshot at a reasonable resolution https://support.apple.com/en-ca/102661

Reply:
awesome thanks